src/OmniSharp.Roslyn/ProjectEventForwarder.cs

```diff
@@ -63,8 +63,6 @@ private void OnWorkspaceChanged(object source, WorkspaceChangeEventArgs args)
                     {
                         Task.Factory.StartNew(async () =>
                         {
-                            await Task.Delay(500);
```

Comment:
😬

Reply:
I don't remember if this was here for a good reason, but that kind of wait is killer when we're trying to make things more responsive.

It would be one thing if we were buffering and sending out the changes grouped together, but in this case we're literally just deferring each event by 500ms.

Reply:
Yes, I've stared at this delay a couple of times before, buy since there are no tests around this, we might easily break VS Code integration 😅 Nobody seems to know why it's there, and why it's 500ms.